content/en/docs/concepts/configuration/secret.md

```diff
@@ -11,18 +11,24 @@ weight: 30
 ---
 
 <!-- overview -->
-
-Kubernetes Secrets let you store and manage sensitive information, such
-as passwords, OAuth tokens, and ssh keys. Storing confidential information in a Secret
-is safer and more flexible than putting it verbatim in a
 {{< glossary_tooltip term_id="pod" >}} definition or in a
```

Comment:
@sftim. I read through the page.
The page looks good. Noted a few nits. This first sentence does not seem right. Would you check?